components/dashboard/Dashboard.js

```diff
@@ -337,18 +360,25 @@ export default function Dashboard(props) {
               {...restProps}
             />
           </Box>
-          <Box className={classes.gridL}>
-            <Card variant="outlined">
-              <CardHeader
-                title="Confidence Level"
-                titleTypographyProps={{ component: 'h2', variant: 'h6' }}
-              />
-
-              <ActivityCategoryTable
-                allActivityLogEntries={allActivityLogEntries}
-                subsetActivityLogEntries={confidentActivityLogEntries}
-                label="Feeling Confident"
-              />
+          <Box className={classes.gridL} position="relative">
+            <RadioButtonUncheckedIcon className={classes.circleIcon} />
+            <TrackChangesIcon className={classes.trackIcon} fontSize="large" />
+            <Card className={classes.card} variant="outlined">
+              <CardContent>
+                <Typography variant="h6" gutterBottom>
+                  Employment Outlook
+                </Typography>
+                <Typography variant="body1" gutterBottom>
+                  Learn about the outlook in your area for the occupation that you want.
+                </Typography>
+              </CardContent>
+              <CardActions>
+                <NextLink href="/employment-outlook">
+                  <Button className={classes.exploreButton} fullWidth onClick={handleClick}>
+                    EXPLORE
+                  </Button>
+                </NextLink>
+              </CardActions>
```

Comment:
Rachel, didn't have these changes in https://github.com/newjersey/career-network/pull/109/commits? Are you repeating it here to build on top of it? Is there possibly a way separating this PR and PR 109 to be as independent as possible from each other? (basically for this PR, have the home dashboard page be the same and have the user navigate to /employment-outlook/ manually). Let me know if you have questions!

Reply:
Great call. yes, it's independent of PR 109. changes removed.